packages/libllvm/build.sh

```diff
@@ -116,19 +114,21 @@ termux_step_post_make_install() {
 	cd $TERMUX_PREFIX/bin
 
 	for tool in clang clang++ cc c++ cpp gcc g++ ${TERMUX_HOST_PLATFORM}-{clang,clang++,gcc,g++,cpp}; do
-		ln -f -s clang-${TERMUX_PKG_VERSION:0:1} $tool
+		ln -f -s clang-${TERMUX_PKG_VERSION:0:2} $tool
 	done
 }
 
 termux_step_post_massage() {
+	# Not added to the package but kept around on the host for other packages like lldb
+	# and rust, which rely on LLVM, to use for configuration.
 	sed $TERMUX_PKG_BUILDER_DIR/llvm-config.in \
 		-e "s|@TERMUX_PKG_VERSION@|$TERMUX_PKG_VERSION|g" \
-		-e "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" \
-		-e "s|@TERMUX_PKG_SRCDIR@|$TERMUX_PKG_SRCDIR|g" \
-		-e "s|@LLVM_TARGET_ARCH@|$LLVM_TARGET_ARCH|g" \
 		-e "s|@LLVM_DEFAULT_TARGET_TRIPLE@|$LLVM_DEFAULT_TARGET_TRIPLE|g" \
-		-e "s|@TERMUX_ARCH@|$TERMUX_ARCH|g" > $TERMUX_PREFIX/bin/llvm-config
```

Comment:
Restored and updated the llvm-config script and noticed that these two `*_ARCH` variables are unused now.